server/controllers/finance/reports/reportAccountsMultiple/report.handlebars

```diff
@@ -0,0 +1,129 @@
+{{> head title="REPORT.REPORTS_MULTIPLE_ACCOUNTS" }}
+
+<body>
+  {{> header }}
+
+  <h3 class="text-center text-uppercase"><strong>{{translate "TREE.REPORTS_MULTIPLE_ACCOUNTS"}}</strong></h3>
+
+  {{#if params.dateFrom }}
+    <h5 style="margin:15px; font-weight:bold" class="text-center text-uppercase">
+      {{date params.dateFrom }} - {{date params.dateTo }}
+    </h5>
+  {{/if}}
+
+  {{#if warnMultipleFiscalYears}}
+  <section style="border : 1px solid black; padding : 5px; margin-bottom : 10px">
+    <p>{{translate "REPORT.REPORT_ACCOUNTS.WARN_MULTIPLE"}}</p>
+  </section>
+  {{/if}}
+
+  {{#unless params.isEnterpriseCurrency}}
+    <div class="alert alert-warning">
+      <p>{{translate "REPORT.REPORT_ACCOUNTS.WARN_CURRENCY"}}</p>
+    </div>
+  {{/unless}}
+
+  <section>
+     {{#each Alltransactions}}
+     <h5><u><b>{{account.number}} | {{translate account.label}}</b></u></h5>
+    <table class="table table-condensed table-report table-bordered">
+      <thead>
+        <tr class="text-capitalize text-center" style="background-color: #ddd;">
+          <th>{{translate "TABLE.COLUMNS.DATE" }}</th>
+          <th>{{translate "TABLE.COLUMNS.TRANSACTION" }}</th>
+          <th>{{translate "TABLE.COLUMNS.DOCUMENT" }}</th>
+          <th>{{translate "TABLE.COLUMNS.DESCRIPTION" }}</th>
+          <th class="text-center">{{translate "FORM.LABELS.RATE" }}</th>
+          <th class="text-center" style="min-width:12%">{{translate "TABLE.COLUMNS.DEBIT" }}</th>
+          <th class="text-center" style="min-width:12%">{{translate "TABLE.COLUMNS.CREDIT" }}</th>
+          <th class="text-center">{{translate "FORM.LABELS.VALUE" }} ({{../currency.symbol}})</th>
+          <th class="text-center" style="min-width:12%">{{translate "TABLE.COLUMNS.BALANCE" }} ({{../currency.symbol}})</th>
+        </tr>
+      </thead>
+      <tbody>
+
+        {{! Opening Balance Line }}
+        <tr>
+          <th>{{ date header.date }}</th>
+          <th colspan="3">{{translate "REPORT.OPENING_BALANCE"}}</th>
+
+          <th class="text-right">
+            {{ header.invertedRate }}
+          </th>
+
+          <th class="text-right">
+            {{currency header.debit ../metadata.enterprise.currency_id}}
+          </th>
+
+          <th class="text-right">
+            {{currency header.credit ../metadata.enterprise.currency_id }}
+          </th>
+
+          <th></th>
+
+          <th class="text-right">
+            {{debcred header.exchangedBalance ../params.currency_id }}
```

Comment:
For some reason, this doesn't reflect the same as the normal account statement in the "balance" column.  Can you check what is going on?

![simpleaccountstatement](https://user-images.githubusercontent.com/896472/46025343-9fbd7100-c0e0-11e8-8fe4-9e30227016a6.PNG)
_This version works_

![multipleaccountsbalance](https://user-images.githubusercontent.com/896472/46025359-a9df6f80-c0e0-11e8-9cc3-d68a062ff4a5.PNG)
_This version isn't correct_